standalone/alertAndPluginDetails.js

```diff
@@ -1,7 +1,7 @@
 /* Output sample:
- *     Alert_Name	Source:PluginName	WASC	CWE
- *     Cross Site Scripting (DOM Based)	ACTIVE:Cross Site Scripting (DOM Based)	8	79
- *     Non-Storable Content	PASSIVE:Content Cacheability	13	524
+ *     Alert_Name    Source:PluginName   WASC    CWE
+ *     Cross Site Scripting (DOM Based) ACTIVE:Cross Site Scripting (DOM Based) 8   79
+ *     Non-Storable Content PASSIVE:Content Cacheability    13  524
```

Comment:
Spotless seems to have changed this from tabs to spaces, though it was supposed to be an example of the output (Which is tab separated)

Reply:
I'm not seeing changes here after running `spotlessApply`, worth reverting these lines.

Reply:
Weird, fixed